src/paperless/checks.py

```diff
@@ -127,15 +127,26 @@ def _ocrmypdf_settings_check():
                 Error(f'OCR output type "{settings.OCR_OUTPUT_TYPE}" is not valid'),
             )
 
-        if settings.OCR_MODE not in {
-            "force",
-            "skip",
-            "redo",
-            "skip_noarchive",
-            "skip_neverarchive",
-        }:
+        if settings.OCR_MODE not in {"force", "skip", "redo", "skip_noarchive"}:
             msgs.append(Error(f'OCR output mode "{settings.OCR_MODE}" is not valid'))
 
+        if settings.OCR_MODE == "skip_noarchive":
```

Comment:
It's minor, but also should be quick to add a couple tests in src\paperless\tests\test_checks.py to hit these lines

Reply:
Good idea. I just pushed a commit to add tests for these.